### PR TITLE
Add support for maintenance check

### PR DESCRIPTION
**Pull request by hepsimo**
Add support for maintenance check

BI-13454

### CODE REVIEW

**Review by PhilipGlover — COMMENTED** [review]
- have used with private-sdk and web changes
- looks good to me
- just need the sdk version updated when we have it
- the Readme ought to be updated with the new config values and maybe an example for British Summer Time

File: pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
         <java.version>1.8</java.version>
         <java-session-handler.version>2.2.0</java-session-handler.version>
-        <private-api-sdk-java.version>2.0.159</private-api-sdk-java.version>
+        <private-api-sdk-java.version>unversioned</private-api-sdk-java.version>
```
Comment (SteveHicksCH) [inline pom.xml:23]: You need to set this value to a publish to main/main-8

**Review by SteveHicksCH — COMMENTED** [review]
Just one comment

**Review by PhilipGlover — APPROVED** [review]
comment as above, and have seen the private-sdk main-8 release specified

**Pull request merged**